Sources/Slacket/Models/Command.swift

```diff
@@ -1 +1,5 @@
-import Foundation
\ No newline at end of file
+import Foundation
+
+protocol SlacketCommandType: SlacketModel {}
+
+//TODO: think do we really need that?
```

Comment:
We ware discussing that we need a way to store incoming slack commands until they are fully processed some kind of stack, that supposed to be a data structure held on that stack. For example if we will get 4 requests one after another from the same user we cannot assume we'll finish processing them in same order (as we're blocked by Parse API and they can have different response times). So when we finally finish processing a request we need a way to mach it with original slack command as it holds an URL where we have to send response.